### PR TITLE
added some italian translation - needs review

### PR DESCRIPTION
**Pull request by cm-schl**
This translation is not complete and needs also some review.
Maybe this could help @lota? 🙂

### CODE REVIEW

**Comment by lota** [comment]
sure @cm-schl I take a look, anyway not seems bad ;-) do you use automatic translator or a real person?

File: snappymail/v/0.0.0/app/localization/it-IT/user.json [hunk 1]
```diff
@@ -405,13 +405,13 @@
...
-		"ALLOW_STYLES": "Allow <style> CSS",
+		"ALLOW_STYLES": "Consenti formattazioni estese all'interno del messaggio (CSS <style>)",
 		"MESSAGE_PER_PAGE": "Messaggi per pagina",
-		"CHECK_MAIL_INTERVAL": "Check mail interval",
+		"CHECK_MAIL_INTERVAL": "Intervallo di aggiornamento della casella postale",
 		"MARK_MESSAGE_READ_AFTER": "Contrassegna il messaggio come letto dopo",
-		"MSG_DEFAULT_ACTION": "Default action",
-		"SECONDS": "seconds",
-		"MINUTES": "minutes",
+		"MSG_DEFAULT_ACTION": "Azione standard",
```
Comment (lota) [inline snappymail/v/0.0.0/app/localization/it-IT/user.json:412]: Azione di default

**Review by lota — CHANGES_REQUESTED** [review]
22: Sposta messagg(i) nel spam
26: Ricordami
29: Salvare le modifiche?
182: Sempre, se possibile
236: Sei sicuro di voler chiudere?

**Comment by cm-schl** [comment]
Thanks 🙂!
A real person 😉 but it's hard to find the correct words if someone has already began to translate - otherwise we would end up in a mix that would seem not logical to the end user 🙂

**Comment by lota** [comment]
22: you cannot use "messagg(i) " because in italian

singolar is: *messaggio*
plural is: *messaggi*

I suppose the canonical lemma should be *messaggio/i*

410: without *?* not a question


for me the a concise lemma is:

*Controlla nuove mail ogni*

(but is not literal translation, but better...)

Alessandro

**Comment by cm-schl** [comment]
Hi,
right, therefore I thought it would be correct to say `messaggi(o)` but you're right with `messaggio/i`. So I will use that instead of `messaggi(o)`.
For 410 your perfectly right, that's a lot more clearer 👍

**Comment by the-djmaze** [comment]
> A real person wink but it's hard to find the correct words if someone has already began to translate - otherwise we would end up in a mix that would seem not logical to the end user

You can also check below link if that helps
https://snappymail.eu/translate.php?lang=it-IT

**Comment by cm-schl** [comment]
> > A real person wink but it's hard to find the correct words if someone has already began to translate - otherwise we would end up in a mix that would seem not logical to the end user
> 
> You can also check below link if that helps https://snappymail.eu/translate.php?lang=it-IT

Oh man, is this a new tool? Very helpful, didn't knew that this exists 🤦‍♂️

**Comment by simonedalpont** [comment]
Hi,
I added some italian translations using your tool

[snappymail-it-IT.zip](https://github.com/the-djmaze/snappymail/files/11287213/snappymail-it-IT.zip)

let me know if you have any doubts about my translations. :)

**Pull request merged**